src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -70,4 +78,19 @@ public class QuicListenerOptions
         public long MaxUnidirectionalStreams { get => throw null; set => throw null; }
         public TimeSpan IdleTimeout { get => throw null; set => throw null; }
     }
+    public class QuicException : Exception
+    {
+        public QuicException(string message) : base(message) { }
+    }
+    public class QuicConnectionAbortedException : QuicException
+    {
+        public QuicConnectionAbortedException(string message, long error) : base(message) { }
```

Comment:
```suggestion
        public QuicConnectionAbortedException(string message, long errorCode) : base(message) { }
```

Just to be consistent.